MobileCenterDistribute/MobileCenterDistribute/Internals/Sender/MSDistributeSender.m

```diff
@@ -12,15 +12,18 @@ @implementation MSDistributeSender
  */
 static NSString *const kMSLatestReleaseApiPathFormat = @"/sdk/apps/%@/releases/latest";
 
-- (id)initWithBaseUrl:(NSString *)baseUrl appSecret:(NSString *)appSecret updateToken:(NSString *)updateToken {
+- (id)initWithBaseUrl:(NSString *)baseUrl
+            appSecret:(NSString *)appSecret
+          updateToken:(NSString *)updateToken
+         queryStrings:(NSDictionary *)queryStrings {
   if ((self = [super initWithBaseUrl:baseUrl
                              apiPath:[NSString stringWithFormat:kMSLatestReleaseApiPathFormat, appSecret]
                              headers:@{
                                kMSHeaderUpdateApiToken : updateToken
                              }
-                        queryStrings:nil
+                        queryStrings:queryStrings
                         reachability:[MS_Reachability reachabilityForInternetConnection]
-                      retryIntervals:@[ @(10) ]])) {
```

Comment:
Yes! I was supposed to remove this to be consistent with Android. Just found that Android has same retry intervals as ingestion, changing this.